src/Illuminate/Collections/Arr.php

```diff
@@ -1059,7 +1059,7 @@ public static function string(ArrayAccess|array $array, string|int|null $key, ?s
     /**
      * Conditionally compile classes from an array into a CSS class list.
      *
-     * @param  array  $array
+     * @param  array|string  $array
```

Comment:
You could narrow the type even further:
```suggestion
     * @param  string[]|array<string, boolean>|string  $array
```
or
```suggestion
     * @param  list<string>|array<string, boolean>|string  $array
```
or
```suggestion
     * @param  array<int, string>|array<string, boolean>|string  $array
```